src/main/java/com/autotune/analyzer/experiment/ExperimentInterfaceImpl.java

```diff
@@ -36,6 +38,7 @@ public class ExperimentInterfaceImpl implements ExperimentInterface {
     public boolean addExperimentToLocalStorage(Map<String, KruizeObject> mainKruizeExperimentMap, List<KruizeObject> kruizeExperimentList) {
         kruizeExperimentList.forEach(
                 (kruizeObject) -> {
+                    LOGGER.info("kruizeObject = {}", kruizeObject.toString());
```

Comment:
Need to make sure to convert this to a `LOGGER.debug`

Reply:
Done